experiments/AMIP/components/atmosphere/climaatmos_extra_diags.jl

```diff
@@ -80,3 +83,95 @@ CAD.add_diagnostic_variable!(
 
     end,
 )
+
+CAD.add_diagnostic_variable!(
+    short_name = "mass_strf",
+    long_name = "Meridional Mass Streamfunction = 2πa cos(lat) ∫_0^z ρv dz",
+    standard_name = "meridional_mass_streamfunction",
+    units = "kg m^-1 s^-1",
+    compute! = (out, state, cache, time) -> begin
+        FT = eltype(state)
+        lat = CC.Fields.coordinate_field(axes(state.f)).lat
+        ρv = CC.Geometry.UVVector.(state.c.uₕ).components.data.:2 .* state.c.ρ
+        ᶠ∫_0_z_ρv = cache.scratch.ᶠtemp_scalar
+        CC.Operators.column_integral_indefinite!(ᶠ∫_0_z_ρv, ρv)
+        twoπa = FT(2π * 6371e3)
+        if isnothing(out)
+            return ᶠ∫_0_z_ρv .* twoπa .* cosd.(lat)
+        else
+            out .= ᶠ∫_0_z_ρv .* twoπa .* cosd.(lat)
+        end
+    end,
+)
+
+CAD.add_diagnostic_variable!(
+    short_name = "stab",
+    long_name = "Static Stability: N = √(g/θ dθ/dz)",
+    standard_name = "static_stability",
+    units = "s^-1",
+    compute! = (out, state, cache, time) -> begin
+        if isnothing(out)
+            return copy(static_stability(cache))
+        else
+            out .= static_stability(cache)
+        end
+    end,
+)
+
+CAD.add_diagnostic_variable!(
+    short_name = "vt",
+    long_name = "Product of meridional wind and temperature",
+    standard_name = "vt",
+    units = "K m s^-1",
+    compute! = (out, state, cache, time) -> begin
+        thermo_params = CAP.thermodynamics_params(cache.params)
+        T = TD.air_temperature.(thermo_params, cache.precomputed.ᶜts)
+        v = CC.Geometry.UVVector.(state.c.uₕ).components.data.:2
+        if isnothing(out)
+            return v .* T
+        else
+            out .= v .* T
+        end
+    end,
+)
+
+CAD.add_diagnostic_variable!(
+    short_name = "egr",
+    long_name = "(zonal) max. Eady growth rate = 0.31 f/N du/dz",
+    standard_name = "max_eady_growth_rate",
+    units = "s^-1",
+    compute! = (out, state, cache, time) -> begin
+        Ω = CAP.Omega(cache.params)
+        φ = CC.Fields.coordinate_field(state.c).lat
+        u = CC.Geometry.UVVector.(state.c.uₕ).components.data.:1
+        du_dz = cache.scratch.ᶜtemp_scalar
+        @. du_dz = CC.Geometry.WVector.(ᶜgradᵥ_.(ᶠinterp_(u))).components.data.:1
+        egr = cache.scratch.ᶜtemp_scalar_2
+        parent(egr) .= parent(FT(0.31) .* 2Ω .* sind.(φ) ./ static_stability(cache) .* du_dz)
+        if isnothing(out)
+            return copy(egr)
+        else
+            out .= egr
+        end
+    end,
+)
+
+"""
+    static_stability(cache)
+
+This function computes the static stability, or the squared root of the Brunt-Väisälä
+frequency N = √(g/θ dθ/dz) using the precomputed temperature field.
+"""
+function static_stability(cache)
+    thermo_params = CAP.thermodynamics_params(cache.params)
+    ᶜts = cache.precomputed.ᶜts
+    θ_virt = TD.virtual_pottemp.(thermo_params, ᶜts)
```

Comment:
Could you add small comments explaining what `θ_virt` and `Nsq` represent? I'm sure it's defined somewhere in atmos, but it would be nice to see it here without having to go digging